### ISSUE TITLE
Login: Allow logging in by site address

### ISSUE THREAD

**Issue by escapemanuele** [Automattic/wp-calypso#83359]
Many users could remember their domain, but not the user/email they used to create it.

We could allow them to login by asking them for their site and then emailing a login link to the linked email.  We can maybe show a censored email like h******@g*******.

### Multiple users

We could maybe show radio button list of all the user emails (hidden like you mentioned h******@g******)? If a site has many users, then showing a long list of radio buttons wouldn't look great visually. Maybe we disallow this login if a site has more than five users, and instead ask them to type their email?

**Comment by niranjan-uma-shankar** [comment]
I like this idea! Curious how the case where a site has multiple users will be handled?

**Comment by escapemanuele** [comment]
I added that to the things to consider @niranjan-uma-shankar! 
What do you think should happen? Is there a primary email associated with a site or just multiple users associated?

**Comment by niranjan-uma-shankar** [comment]
@escapemanuele A site could have many users with different [roles](https://wordpress.org/documentation/article/roles-and-capabilities/). We can't know which of these users are attempting to log in with the site address. 



We could maybe show radio button list of all the user emails (hidden like you mentioned h******@g******)? If a site has many users, then showing a long list of radio buttons wouldn't look great visually. Maybe we disallow this login if a site has more than five users, and instead ask them to type their email?

**Comment by escapemanuele** [comment]
Seems reasonable @niranjan-uma-shankar, I updated the issue description

**Comment by alshakero** [comment]
We decided to move on from this issue for the meantime. I learned a lot, and here's all the relevant resources for the next person.

Start with: p5TWut-114-p2
[Attempt I](https://github.com/Automattic/wp-calypso/pull/84391) (bad attempt): email the site owner a magic link using their site URL.
[Attempt II](https://github.com/Automattic/wp-calypso/pull/84606) (better attempt): allow site URL + password combinations for sites that have one owner. 

Both of the Calypso PRs link to Phab patches showing the needed backend changes.
